src/plugins/es_ui_shared/static/forms/helpers/field_validators/index_name.ts

```diff
@@ -17,14 +17,12 @@
  * under the License.
  */
 
-// Note: we can't import from "ui/indices" as the TS Type definition don't exist
-// import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from 'ui/indices';
+// @ts-ignore
+import { INDEX_ILLEGAL_CHARACTERS_VISIBLE } from '../../../../indices';
```

Comment:
@sebelga AFAICT this file isn't consumed anywhere, so I wasn't sure how to test this. I don't want to spend time writing a test for it. I'm inclined to leave it with the expectation that a problem will be surfaced and fixed if/when we consume this file. What are your thoughts?